front/lib/models.ts

```diff
@@ -1241,6 +1241,200 @@ DataSource.hasMany(DocumentTrackerChangeSuggestion, {
   onDelete: "CASCADE",
 });
 
+// Assistant V2
+export class Conversation extends Model<
+  InferAttributes<Conversation>,
+  InferCreationAttributes<Conversation>
+> {
+  declare id: number;
+  declare sId: string;
+  declare title: string;
+  declare created: Date;
+  declare visibility: ChatSessionVisibility;
+}
+
+Conversation.init(
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      autoIncrement: true,
+      primaryKey: true,
+    },
+    sId: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      unique: true,
+    },
+    title: {
+      type: DataTypes.TEXT,
+      allowNull: false,
+    },
+    created: {
+      type: DataTypes.DATE,
+      allowNull: false,
+    },
+    visibility: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      defaultValue: "private",
+    },
+  },
+  {
+    modelName: "conversation",
+    sequelize: front_sequelize,
+  }
+);
+
+export class UserMessage extends Model<
+  InferAttributes<UserMessage>,
+  InferCreationAttributes<UserMessage>
+> {
+  declare id: number;
+  declare textContent: string;
+  declare userId: ForeignKey<User["id"]>;
+}
+
+UserMessage.init(
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      autoIncrement: true,
+      primaryKey: true,
+    },
+    textContent: {
```

Comment:
would call it message
we also need to add:

userContextUsername (required)
userContextTimezone (required)
userContextFullName (optional)
userContenxtEmail (optional)
userContextProfilePictureUrl (optional)

Not sure if we want to linearize everything or use a postgres JSON for that?